lib/animas/animasSimulator.js

```diff
@@ -167,24 +167,25 @@ exports.make = function(config){
       }
 
       if(currStatus !== null) {
-        if(currStatus.status === 'suspended') {
```

Comment:
I'm having some trouble reading this logic, but it's probably just a *me* problem. So the reason you've deleted the check against `currStatus.status` is because the *only* statuses that exist now are `suspended`s, right?